backend/api/serializers/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .register_serializer import RegisterSerializer as RegisterSerializer
 from .login_serializer import LoginSerializer as LoginSerializer
 from .community_serializer import CommunitySerializer as CommunitySerializer
+from .feedback_serializer import FeedbackSerializer
```

Comment:
use 'as FeedbackSerializer' - suppresses warnings of unused import (for linting)